nimbus/src/evaluator.rs

```diff
@@ -71,7 +71,7 @@ pub fn evaluate_enrollment(
         match available_randomization_units
```

Comment:
We no longer need this match - ie, this line can read `let id = available_randomization_units.get_value(...);`

Reply:
Fixed.